sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md

```diff
@@ -1,34 +1,37 @@
-# 4.1.1
-## Bug fixes
+Release History
```

Comment:
Use `#` heading.

Reply:
Thanks. Fixed in this PR https://github.com/Azure/azure-sdk-for-net/pull/9168/files